mdtraj/geometry/tests/test_hbonds.py

```diff
@@ -145,3 +145,24 @@ def test_wernet_nilsson_1():
             # to make sure the criterion is giving back totally implausible stuff
             if len(hbonds) > 0:
                 assert np.all(md.compute_distances(t[frame], hbonds[:, [0,2]]) < 0.5)
+def test_water_solute():
```

Comment:
Should you test or donor-acceptor selections? I assume you can manually calculate how long the donors and acceptors lists should be.